### PR TITLE
add api for get random tweet

### PR DESCRIPTION
**Pull request by sjchin88**

### CODE REVIEW

File: services/data_service.py [hunk 1]
```diff
@@ -45,3 +49,20 @@ def generate_word_cloud(self, model_id: int) -> List[WordResponse]:
...
+    def read_random(self)->Data:
+        # id_list: List[int] = []
+        # all_datas = Data.query.all()
+        # for datum in all_datas:
+        #     id_list.append(datum.id)
+        # #print(id_list)
+        # random_id = random.choice(id_list)
+        # #print(random_id)
+        # random_data = self.read_by_id(random_id)
+        # #print(random_data)
```
Comment (zacherysmith115) [inline services/data_service.py:62]: Can probably get rid of these

File: services/data_service.py [hunk 2]
```diff
@@ -45,3 +49,20 @@ def generate_word_cloud(self, model_id: int) -> List[WordResponse]:
...
+        starttime = timeit.default_timer()
+        random_data = Data.query.order_by(func.random()).first()
+        #print(random_data)
```
Comment (zacherysmith115) [inline services/data_service.py:65]: Same here

File: services/data_service.py [hunk 3]
```diff
@@ -45,3 +49,20 @@ def generate_word_cloud(self, model_id: int) -> List[WordResponse]:
...
+        print(timeit.default_timer()-starttime)
```
Comment (zacherysmith115) [inline services/data_service.py:66]: Probably can remove the timing function and print

File: services/data_service.py (hunk 1 above)
Comment (sjchin88) [reply services/data_service.py]: done

File: services/data_service.py (hunk 3 above)
Comment (sjchin88) [reply services/data_service.py]: done

**Review by edwardlach — APPROVED** [review]

**Pull request merged**